test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -146,4 +173,10 @@ public static boolean isAwsExecution() {
         return dockerConfig.trim().equalsIgnoreCase("true") ?  true : false;
 
     }
+
```

Comment:
we can change the access level to private.

Reply:
Why? That will break calls to this method from other classes.
This method was not added in this PR.

Reply:
The comment was for the method `isSecurityEnabled()`, we can change the access level to private